spec/README.md

```diff
@@ -0,0 +1,22 @@
+Jakarta WebSocket Specification
+============================
+
+This project generates the Jakarta WebSocket Specification.
+
+Building
+--------
+
+Prerequisites:
+
+* JDK8+
+* Maven 3.0.3+
```

Comment:
Maven 3.0.x is a strange choice.
Eg: if you use Java 9+ then you'll have issues.